### PR TITLE
Light mode form modal dialogs

### LINKED ISSUE #7184: Light mode for modal dialogs
### Is your enhancement related to a problem? Please describe

Modal dialogs do not support light mode.

### Describe the solution you'd like

Extract colors to the registry and add light mode colors. The component structure is being cleaned up, but we possibly have several of these that need to be moved. e.g. Modal, MessageBox, making sure things like the Feedback form and Container > Create are fixed.

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by feloy**
Signed-off-by: Philippe Martin <phmartin@redhat.com>

### What does this PR do?

Depends on #7315 

### What issues does this PR fix or reference?

Fixes #7184

### How to test this PR?

- [x] Tests are covering the bug fix or the new feature

### CODE REVIEW

File: packages/renderer/src/lib/container/ContainerList.svelte [hunk 1]
```diff
@@ -690,16 +690,17 @@ function setStoppedFilter() {
     }}">
     <div
       role="presentation"
-      class="inline-block w-full overflow-hidden text-left transition-all transform bg-charcoal-600 z-50 rounded-xl shadow-xl shadow-charcoal-900"
+      class="inline-block w-full overflow-hidden text-left transition-all transform z-50 rounded-xl"
```
Comment (deboer-tim) [inline packages/renderer/src/lib/container/ContainerList.svelte]: You should be able to remove extraneous z-50 and rounded-xl as well.

File: packages/main/src/plugin/color-registry.ts [hunk 2]
```diff
@@ -785,13 +785,41 @@ export class ColorRegistry {
...
+      light: colorPalette.gray[50],
     });
     this.registerColor(`${modal}border`, {
       dark: colorPalette.charcoal[500],
-      light: colorPalette.gray[200],
+      light: colorPalette.gray[500],
+    });
+    this.registerColor(`${modal}header-bg`, {
+      dark: colorPalette.black,
+      light: colorPalette.gray[100],
```
Comment (deboer-tim) [inline packages/main/src/plugin/color-registry.ts:806]: The difference between header and modal bg is almost unnoticeable in light mode for me (but can always be tweaked later).

**Review by deboer-tim — APPROVED** [review]
A couple minor comments, but LGTM.

**Comment by feloy** [comment]
I have made changes to integrate the changes done in #7315

**Review by deboer-tim — APPROVED** [review]

**Review by benoitf — APPROVED** [review]

**Pull request merged**